tendrl/commons/objects/node_context/__init__.py

```diff
@@ -29,9 +30,17 @@ def __init__(self, machine_id=None, node_id=None, fqdn=None,
         self._etcd_cls = _NodeContextEtcd
 
     def _get_machine_id(self):
-        cmd = cmd_utils.Command("cat /etc/machine-id")
-        out, err, rc = cmd.run()
-        return str(out)
+        try:
+            out = None 
+            with open('/etc/machine-id', 'r') as f:
+                out = f.read().strip('\n') 
+        except IOError as ex:
+            exc_type, exc_value, exc_tb = sys.exc_info()
+            traceback.print_exception(
+                exc_type, exc_value, exc_tb, file=sys.stderr)
+            sys.stderr.write(
```

Comment:
Message class required node_context to log the message, so we can't log this exception using message class